src/go/rpk/pkg/cli/debug/common/common.go

```diff
@@ -0,0 +1,50 @@
+package common
+
+import (
+	"strings"
+	"time"
+
+	"github.com/spf13/pflag"
+)
+
+// DebugBundleSharedOptions are the options of a debug bundle that can be
+// shared between a normal and a remote bundle.
+type DebugBundleSharedOptions struct {
+	CPUProfilerWait         time.Duration
+	LogsSince               string
+	LogsUntil               string
+	MetricsInterval         time.Duration
+	MetricsSampleCount      int
+	PartitionFlag           []string
+	Namespace               string
+	LabelSelector           []string
+	LogsSizeLimit           string
+	ControllerLogsSizeLimit string
+}
+
+// InstallFlags installs the debug bundle flags that fills the debug bundle
+// options.
+func (o *DebugBundleSharedOptions) InstallFlags(f *pflag.FlagSet) {
+	f.StringVar(&o.ControllerLogsSizeLimit, "controller-logs-size-limit", "132MB", "The size limit of the controller logs that can be stored in the bundle (e.g. 3MB, 1GiB)")
+	f.DurationVar(&o.CPUProfilerWait, "cpu-profiler-wait", 30*time.Second, "For how long to collect samples for the CPU profiler (e.g. 30s, 1.5m). Must be higher than 15s")
+	f.StringVar(&o.LogsSizeLimit, "logs-size-limit", "100MiB", "Read the logs until the given size is reached (e.g. 3MB, 1GiB)")
+	f.StringVar(&o.LogsSince, "logs-since", "yesterday", "Include logs dated from specified date onward; (journalctl date format: YYYY-MM-DD, 'yesterday', or 'today'). Refer to journalctl documentation for more options")
```

Comment:
```suggestion
	f.StringVar(&o.LogsSince, "logs-since", "yesterday", "Include logs dated from specified date onward. For example: journalctl date format: YYYY-MM-DD, 'yesterday', or 'today'. See the journalctl documentation for more options.")
```

Reply:
What do you think of leaving just: `journalctl date format` as is the only option, and not an example of how to pass the flag?